src/components/Tile/Tile.md

```diff
@@ -91,3 +91,37 @@ const image = <Image source={yammerLogo} description={yammerLogoDescription} ful
   </FixedGridRow>
 </div>
 ```
+
+With border color:
+
+```js { "props": { "data-description": "with border color" } }
+const { TileAspectRatio } = require('.');
+const { ImageFit } = require('../Image');
+const { TextColor } = require('../Text');
+
+const yammerLogo = 'logo.png';
+const yammerLogoDescription = 'Yammer "y" logo';
+
+const image = <Image source={yammerLogo} description={yammerLogoDescription} fullWidth={true} />;
```

Comment:
It changes depending on how zoomed in I am

Reply:
Interesting, I would expect whitespace to not exist when using imageFit.cover. Is this a side effect of using the 1px top/left/right/bottom values?

Reply:
It isn't always there. It depends on my zoom for some reason. At different zoom percentages, different tiles show the whitespace and sometimes none of them have this problem.

Reply:
I just spoke to @BrendanMcK and it looks like an issue with ImageFit and how it transforms the image. Most likely a pixel rounding issue because the width and height are odd here. I am going to log an issue for Image. 

So it seems like it's not an issue with tile. Is it ok to merge this PR then?

Reply:
I think it's probably okay to go ahead with merging in that case. @chrismohr @swese44 what you think?